internal/nexodus/bootstrap_dns.go

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) Tailscale Inc & AUTHORS
```

Comment:
Anywhere we are importing code that we don't intend to modify, I would prefer to keep it out of `internal/nexodus`. You could put it in `pkg/` where we have similar code that should remain decoupled from `nexd`.

Related: it would be good to have a developer doc that describes how to sync this code with its upstream.

Reply:
sure, I will work on it.